src/patternfly/components/Sequence/docs/code.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
+
+For example for buttons: Always add a modifier class to add color to the button. Never use the class `.btn` on its own.
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[landmark description]"` | `.pf-c-sequence` | Describes <sequence> landmark |
+| `aria-setsize` | `.pf-c-sequence__item` | Indicates the total number of items in the list |
+| `aria-posinset` | `.pf-c-sequence__item` | Indicates the position of an item within a list |
+| `aria-hidden="true"` | `.pf-c-sequence__control-icon` | Hides the icon from assistive technologies |
+| `aria-label="actions"` | `.pf-c-sequence__control-icon` | Provides an accessible name for the dropdown when an icon is used instead of text. Required when an icon is used with no supporting text |
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-sequence` | `<div>` |  Initiates a primary sequence element |
+| `.pf-c-sequence__list` | `<ul>` | Initiates a default sequence list |
+| `.pf-c-sequence__item` | `<li>` | Initiates a default sequence list item |
+| `.pf-c-sequence__link` | `<a>` | Initiates a default sequence list link |
+| `.pf-c-sequence__number-container` | `<div>` | Initiates a sequence number container element |
+| `.pf-c-sequence__number` | `<div>` | Initiates a sequence number element |
+| `.pf-c-sequence__content` | `<div>` | Initiates a sequence content element |
+| `.pf-c-sequence__content-title` | `<h1>, <h2>, <h3>, <h4>, <h5>, <h6>` | Initiates a sequence content title |
```

Comment:
this is applied to a `span` in your examples.

Reply:
changed